crates/driver/src/infra/solver/mod.rs

```diff
@@ -48,6 +48,16 @@ pub struct Slippage {
     pub absolute: Option<eth::Ether>,
 }
 
+#[derive(Clone, Copy, Debug)]
+pub enum Liquidity {
```

Comment:
Curious what your mental model is for when to use a two type enum vs a boolean directly (comes up in #1926 as well)

Reply:
Here it was that I find `solver::Liquidity::Skip` more descriptive than `false` as a value. In fact, it makes the configuration value not require additional context to makes sense (`solver::Liquidity::Skip` obviously means to skip liquidity for the solver, while `false` can be `skip_liquidity: false` or `fetch_liquidity: false`, so the added "where the value comes from" matters). Another tiny argument for this would be that this isn't necessarily a true/false value, but can become more complex, and expanded to convey more information:

```
enum Liquidity {
    Skip,
    Fetch,
    OnlyKinds {
        kinds: Vec<liquidity::Kind>,
    },
}
```

Obviously, you can go too far with this, but I found here that it made sense. Always happy to discuss and re-evaluate if my mental model is correct or just being pedantic.